environs/config/config.go

```diff
@@ -2023,4 +2027,9 @@ data of the store. (default false)`,
 		Type:        environschema.Tstring,
 		Group:       environschema.EnvironGroup,
 	},
+	DefaultSpace: {
+		Description: "Default Spaces",
```

Comment:
"The default network space used for application endpoints in this model"

Reply:
:+1: